pages/styles/article.module.css

```diff
@@ -0,0 +1,3 @@
+.article {
+  text-align: left;
```

Comment:
Let's also do:

```
display: flex;
flex-direction: column;
width: 100%;
```

When you change it to just be a div, this will mean the whole markdown article will have every separated element be an item in the flex container 🥳

Reply:
yeah sure thing!